admin-dev/themes/new-theme/js/components/form/form-object-mapper.ts

```diff
@@ -451,15 +460,21 @@ export default class FormObjectMapper {
    * This method is called when this component initializes or when triggered by an external event.
    */
   private updateFullObject():void {
-    const serializedForm = this.$form.serializeJSON({
-      checkboxUncheckedValue: '0',
+    // Temporarily enable all inputs or they will not be serialized
```

Comment:
Not that much because we use exactly the same found input to disable and enable them, so we are sure that everything that was enabled will be disabled again

The risk would be on element which dynamically change the disable mode, but I tested it with some of our disabled elements in the product page that are disabled via a switch and even those work fine